Gemfile

```diff
@@ -50,6 +56,12 @@ group :test do
   gem 'selenium-webdriver'
   # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
+  #additional gem files for RSpec testing 
+  gem 'rspec-rails'
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'rails-controller-testing'
+  gem 'rexml'
```

Comment:
You have doubled up your gems, having the same ones listed in `group :development` and `group :test`. I _think_ you should really only need these gems for the test environment, but if you do need them for both you can cut down on the duplication by putting them under a new group like this:

```rb
group :development, :test do
  gem 'rspec-rails'
  gem 'factory_bot_rails'
end
```

Then it will install them for both environments.